examples/server/tests/features/steps/steps.py

```diff
@@ -1045,7 +1056,17 @@ def start_server_background(context):
     if 'SERVER_LOG_FORMAT_JSON' not in os.environ:
         server_args.extend(['--log-format', "text"])
     print(f"starting server with: {context.server_path} {server_args}\n")
+    flags = 0
+    if 'nt' == os.name:
+        flags |= 0x00000008  # DETACHED_PROCESS
+        flags |= 0x00000200  # CREATE_NEW_PROCESS_GROUP
+        flags |= 0x08000000  # CREATE_NO_WINDOW
+
+    pkwargs = {
+        'close_fds': True,  # close stdin/stdout/stderr on child
```

Comment:
The comment here is incorrect - close_fds closes all file descriptors *except* for stdin/stdout/stderr. It also happens to be True by default anyway, so setting it to True here is redundant.